requirements.txt

```diff
@@ -26,18 +26,17 @@ git+https://github.com/dotkom/django-guardian.git@1b184d5377200ab4ad59b343ae8af9
 django-taggit==0.22.2           # Generic multi-model tagging library
 django-taggit-serializer==0.1.5 # REST Framework serializers for Django-taggit
 APScheduler==3.5.1              # Scheduler
-feedme==1.9.5
-git+https://github.com/dotKom/redwine.git@0690807537fcbeeacde9c40b0e45549e0d216a96#egg=redwine==1.2.4.1
+git+https://github.com/dotkom/redwine.git@c26e12003a26bd4ef15c10d55358c24f96b3d132#egg=redwine
```

Comment:
A version should be provided otherwise I think pip will never upgrade the package once downloaded.
I think the previous version was fake since I (or whoever upgraded it last time) was too lazy to update the redwine version.

Reply:
Defined a version, and made a release of Redwine as well, just for the fun of it.